apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Supabase Integrations Marketplace'
```

Comment:
let's go with:
"Supabase Integrations" 
or
"Supabase Integrations v2.0" 
@kiwicopple wyt?

Reply:
or maybe "Supabase Integrations v1.0" based on the first sentence "we've been running our [Integrations Marketplace](/partners) in “stealth mode” for about a year now"